objects/client.c

```diff
@@ -2153,13 +2161,40 @@ client_set_icon_from_pixmaps(client_t *c, xcb_pixmap_t icon, xcb_pixmap_t mask)
 
 /** Kill a client.
  *
+ * By default, this will use the normal X11 way of closing a client (the `X`
+ * button). If a signal number is provided, it will be used instead.
+ *
+ * This can be used to pause or wakeup some clients or to save energy.
```

Comment:
> This can be used to pause or wakeup some clients or to save energy.

Better:

> This can be used to pause or wakeup some clients, which can be used to save energy.